### ISSUE TITLE
Remove/modify existing MUTO-INTL samples in `submission-song`

### ISSUE THREAD

**Issue by edsu7** [icgc-argo/workflow-roadmap#338]
JingWei attempted to submitted samples but was blocked by song due to :
```
SONG_SERVER_ERROR[mismatching.specimen.data @ 1679929966039]: [CompositeEntityService::mismatching.specimen.data] - Input Specimen data does not match the existing Specimen data for submitterSpecimenId 'PD37727b'. Ensure the data matches.
```
In other words the current metadata doesn't match old "cached" metadata in song.

The current metadata:
https://clinical.platform.icgc-argo.org/clinical/program/MUTO-INTL/donor/DO250010
```
    {
      "samples": [
        {
          "sampleType": "Total DNA",
          "sampleId": "SA610023"
        }
      ],
      "specimen": {
      "specimenTissueSource": "Blood derived - peripheral blood",
      "tumourNormalDesignation": "Normal",
      "specimenType": "Normal",
      "specimenId": "SP210020"
      }
    }
```
vs submission-song
https://submission-song.rdpc.cancercollaboratory.org/studies/MUTO-INTL/samples/SA610023
https://submission-song.rdpc.cancercollaboratory.org/studies/MUTO-INTL/specimens/SP210020
```
  "samples": [
    {
      "sampleType": "Amplified DNA",
      "sampleId": "SA610023",
      "specimen": {
        "specimenTissueSource": "Blood derived",
        "tumourNormalDesignation": "Normal",
        "specimenType": "Normal"
        "specimenId": "SP210020",
      },
      }
    }
  ]
```

Will have to delete old samples in SONG

**Comment by edsu7** [comment]
Original ticket suppressing analyses:
https://github.com/icgc-argo/workflow-roadmap/issues/296

Which was blocked by inability to delete samples:
https://github.com/overture-stack/SONG/issues/777

**Comment by edsu7** [comment]
https://docs.google.com/spreadsheets/d/1f8e9CPhGClWruqeCuTrFm-F7tEWT57JE2nPXzuUUvLs/edit?usp=sharing

**Comment by puneet-oicr** [comment]
JingWei to confirm what the right data is at their end. Then Jon/Justin to confirm next steps

**Comment by puneet-oicr** [comment]
Waiting for JingWei to respond. Edmund to send follow up email

**Comment by puneet-oicr** [comment]
JingWei cc'ed a few people, there is input from them, Edmund to look in to it

**Comment by puneet-oicr** [comment]
Create an SOP for this typical modification

**Comment by joneubank** [comment]
Data fixed in Submission.SONG for specimens identified in spreadsheet.

**Comment by joneubank** [comment]
There is an open question for the spreadsheet data, where some donors have only one sample updated to `Amplified DNA` but it seems logical that both should be updated. @edsu7 will confirm if we have the complete set of updates or if more are required. We will update the clinical DB once this is confirmed.

**Comment by puneet-oicr** [comment]
Awaiting Thomas's update.

**Comment by joneubank** [comment]
Confirmed all MUTO-INTL samples to be updated to be `Amplified DNA`. 515 donors updated.

Please test submission now, all data should be corrected.

**Comment by edsu7** [comment]
Emailed Jingwei to inform him changes have been made. He's now unblocked for submission. Will follow up on submission in another ticket.
